comp/metadata/packagesigning/README.md

```diff
@@ -0,0 +1,58 @@
+# Package Signing Payload
+
+This component populates the Linux Packages signing information in the `inventories` product in DataDog. They fill the `host_signing` table.
+
+This is enabled by default but can be turned off using `inventories_enabled` config.
+
+The payload is sent every 12h (see `inventories_max_interval` in the config). It has a different collection timeframe than the other inventory payloads.
+
+# Format
+
+The payload is a JSON dict with a list of keys, each having the following fields
+
+- `hostname` - **string**: the hostname of the agent as shown on the status page.
+- `timestamp` - **int**: the timestamp when the payload was created.
+- `signing_keys` - **list of dict of string to JSON type**
+  - `fingerprint` - **string**: the 16-char long key fingerprint.
+  - `expiration_date` - **string**: the expiration date of the key.
+  - `key_type` - **string**: the type of key. which represents how it is referenced in the host. Possible values are "signed-by", "trusted" or "debsig" for DEB-based distributions, "repo" or "rpm" for RPM-based distributions.
```

Comment:
Can a key have more than one value for the same key? If so, would an `list[str]` be better for deduping?

Reply:
This one is an interesting suggestion (I initially didn't want to store duplicates but it eventually make sense). However it would probably need a redefinition of tables on backend side. Can I take this as a phase 2 improvement?